### DISCUSSION TITLE
Add non-METAL common backend only to PyPI

### DISCUSSION THREAD

**Discussion by kgourgou** [ml-explore/mlx#174] [Q&A] [answered]
Hello and thanks for the great work on the MLX, 

I enjoy writing research code in it, but I'm a bit hesitant to share it as some of my colleagues / friends won't be able to use it due to the ARM dependency. 

Are there any plans for some sort of fallback simulation mode that would run on intel processors / other platforms?

**Comment by awni** [comment, marked as answer]
We actually have a generic back-end that can run on Intel CPUs. Right now it has to be built from source, but we could make a pypi wheel for it. It's not a big priority for us so it may not be as well optimized as getting something platform specific.

We do not have plans to do a CUDA backend or other accelerators in the near future.

**Reply by kgourgou** [reply]
Excellent, that would be sufficient I think. 

Understandable re: CUDA backend, etc.
